params/config.go

```diff
@@ -26,28 +26,32 @@ import (
 var (
 	// MainnetChainConfig is the chain parameters to run a node on the main network.
 	MainnetChainConfig = &ChainConfig{
-		ChainId:        MainNetChainID,
-		HomesteadBlock: MainNetHomesteadBlock,
-		DAOForkBlock:   MainNetDAOForkBlock,
-		DAOForkSupport: true,
-		EIP150Block:    MainNetHomesteadGasRepriceBlock,
-		EIP150Hash:     MainNetHomesteadGasRepriceHash,
-		EIP155Block:    MainNetSpuriousDragon,
-		EIP158Block:    MainNetSpuriousDragon,
-		Ethash:         new(EthashConfig),
+		ChainId:         MainNetChainID,
+		HomesteadBlock:  MainNetHomesteadBlock,
+		DAOForkBlock:    MainNetDAOForkBlock,
+		DAOForkSupport:  true,
+		EIP150Block:     MainNetHomesteadGasRepriceBlock,
+		EIP150Hash:      MainNetHomesteadGasRepriceHash,
+		EIP155Block:     MainNetSpuriousDragon,
+		EIP158Block:     MainNetSpuriousDragon,
+		MetropolisBlock: MainNetMetropolisBlock,
+
+		Ethash: new(EthashConfig),
 	}
 
 	// TestnetChainConfig contains the chain parameters to run a node on the Ropsten test network.
 	TestnetChainConfig = &ChainConfig{
-		ChainId:        big.NewInt(3),
-		HomesteadBlock: big.NewInt(0),
-		DAOForkBlock:   nil,
-		DAOForkSupport: true,
-		EIP150Block:    big.NewInt(0),
-		EIP150Hash:     common.HexToHash("0x41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d"),
-		EIP155Block:    big.NewInt(10),
-		EIP158Block:    big.NewInt(10),
-		Ethash:         new(EthashConfig),
+		ChainId:         big.NewInt(3),
+		HomesteadBlock:  big.NewInt(0),
+		DAOForkBlock:    nil,
+		DAOForkSupport:  true,
+		EIP150Block:     big.NewInt(0),
+		EIP150Hash:      common.HexToHash("0x41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d"),
+		EIP155Block:     big.NewInt(10),
+		EIP158Block:     big.NewInt(10),
+		MetropolisBlock: TestNetMetropolisBlock,
```

Comment:
I'd recommend using the real value, not an intermediate constant here. It'd be cleaner.